cv_next/app/profile/[profileId]/components/profileCvs.tsx

```diff
@@ -0,0 +1,40 @@
+"use client";
```

Comment:
If I understand correctly, `"use client"` is not necessary here (this entire section can be SS-rendered).